Warehouse.Api.Auth/Business/AuthService.cs

```diff
@@ -43,46 +47,51 @@ public async Task<Result<UserDto>> RegisterAsync(RegisterDto register)
             var user = Mapper.Map<UserDto>(register);
             var hashedPassword = _hasher.HashPassword(user, register.Password);
             user = user with {PasswordHash = hashedPassword, Roles = new List<string> {"User"}};
-            await _userService.CreateAsync(user);
+            await _sender.SendMessage(user, "CreateUserQueue");
```

Comment:
It's better to move the string literal to some Constants class (e.g. RabbitMqMessages.cs)